README.md

```diff
@@ -365,102 +365,82 @@ Any data format, such as JSON or raw bytes, can be stored as long as
-filters include:
-
-- `setWriterIds`: Filter to records written by these IDs
-- `setUserIds`: Filter to records with these user IDs
-- `setRecordIds`: Filter to only the records identified by these IDs
-- `setTypes`: Filter to records that match the given types
-- `setIncludeAllWriters`: Set this flag to include records that have been shared
-  with you, defaults to `false`
-
-> While the Java SDK supports _writing_ plaintext meta with records, the query interface does not support filtering on that meta information at this time.
-
-## Pagination
-
-The `QueryResponse` object's method `last()` gives a value indicating the last
-record returned. Passing this value to the `setAfter()` method on the `QueryParamsBuilder`
-object will cause E3DB to return records that come "after" that value. For example, this
-snippet will loop through all "lyric" records in 10-row increments:
-
-```java
-Client client = ...; // Get a client instance
-
-// Create a parameter builder that we can re-use to
-// call the `setAfter()` method over and over, for
-// pagination.
-final QueryParams params = new QueryParamsBuilder()
-  .setTypes("lyric")
-  .setIncludeData(true)
-  .setCount(10);
-
-// Allows us to modify this flag from inside the `ResultHandler`
-// anonymous class below.
-final AtomicReference<Boolean> done = new AtomicReference<>(false);
-
-while(! done.get()) {
-  CountDownLatch wait = new CountDownLatch(1);
-
-  client.query(params.build(), new ResultHandler<QueryResponse>() {
-     @Override
-     public void handle(Result<QueryResponse> r) {
-       if(! r.isError()) {
-         List<Record> records = r.asValue().records();
-         if(records.size() == 0) {
-           // no more records, stop looping
-           done.set(true);
-         }
-         else {
-           // print list of records for this page
-           for(Record r : r.asValue().records()) {
-             System.out.println("Record ID: " + r.meta().recordId());
-             System.out.println("Song: " + r.data().get("song"));
-           }
-
-           // set next page in parameter builder
-           params.setAfter(r.asValue().last());
-         }
-
-         wait.countDown();
-       }
-     }
-   }
-  );
-
-  wait.await(30, TimeUnit.SECONDS);
+public class SearchExample {
+    public static void searchExample(Client client) throws InterruptedException {
+
+        // Create an initial search request.
+        SearchRequest searchRequest = new SearchRequestBuilder().
+                setIncludeData(true).
+                setIncludeAllWriters(false).
+                setLimit(50).
+                // Set a range of the search for records created from 5 days ago until now
+                setRange(new SearchRequest.SearchRange(SearchRequest.SearchRangeType.CREATED,
+                        Date.from(Instant.now().minus(5, ChronoUnit.DAYS)),
+                        null)).
+                setOrder(SearchRequest.SearchOrder.SearchOrderAscending()).
```

Comment:
comment here for how this affects the returned results, I always have to check the API spec myself https://github.com/tozny/api-specs/blob/master/search/swagger.yaml#L268